### DISCUSSION TITLE
How not to use UltraGrid screen when using Spout

### DISCUSSION THREAD

**Discussion by ygreq** [CESNET/UltraGrid#201] [Q&A]
Hi there!

Considering I have a connection that has Spout as video capture device, but more importantly in this situation, it also has Spout as display device, is it possible to make it so that I don't get to have a UltraGrid screen once the connection gets established? What I mean is once UltraGrid gets the image from the other UltraGrid instance, to send the image to other apps via Spout without projecting it on the UltraGrid screen.

This is how the code looks like now: 
`uv -t spout:name=fromMax:fps=60:codec=RGB -c libavcodec:codec=H.264 -d gl:spout=fromUV [IP of other PC]`

Is this possible?

**Comment by MartinPulec** [comment]
Hi, I am not sure if I completely understood your question - basically you mean that you don't want to have the OpenGL window visible? This is pretty much easy, you can use `-d gl:spout=fromUV:hide-window`.

**Reply by ygreq** [reply]
I believe this is what I was searching for. :D I will test asap! Thank you so much!
